src/common/IOSTypes.ts

```diff
@@ -80,9 +91,9 @@ export class IOSSimulatorDevice {
     }
 
     /**
-     * A string representation of an IOSSimulatorDevice which includes Name, Runtime Id
+     * A string representation of an IOSSimulatorDevice which includes Name, Runtime Id, and udid
      */
     public toString(): string {
-        return `${this.name}, ${this.runtimeId}`;
+        return `${this.name}, ${this.runtimeId}, ${this.udid}`;
```

Comment:
If I recall correctly we didn't include `udid` originally b/c we didn't think that would be useful info for the user. Often times the user cares more about human-readable info, so we kept name + runtime info but not `udid`. Are you including it here b/c you want to use it in another place (for example by parsing the output)?

Reply:
The idea @maliroteh-sf is that you will be able to list iOS devices with
sfdx force:lightning:local:device:list -p iOS

And then to start an iOS device and start the exact device you want, you can use:
sfdx force:lightning:local:device:start -p iOS -t "<udid>"

You can still use name:
sfdx force:lightning:local:device:start -p iOS -t "<name>"

But if you do, we will select the first one in the list since the names are not unique for a device with multiple simulators for each version of iOS.  By displaying the udid in the list, you will now be able to choose the specific iOS device to start and therefore be able to start the device with a particular ios version.